web/src/drafts.ts

```diff
@@ -117,8 +118,19 @@ export const draft_model = (function () {
         const parsed_drafts = possibly_buggy_drafts_schema.parse(drafts);
         const valid_drafts: Record<string, LocalStorageDraft> = {};
         for (const [draft_id, draft] of Object.entries(parsed_drafts)) {
-            if (draft.type !== "stream") {
-                valid_drafts[draft_id] = draft;
+            // TODO/compatibility: This can be deleted once [?? TODO(evy) fill this in]
```

Comment:
// TODO/compatibility: We should eventually be able to delete this. But probably not anytime soon. Once you can no longer upgrade to `main` without first upgrading to 11.0, we can be certain clients that have actually logged in have experienced this conversion code... but even after that, a client may still have old-style drafts for several months.

Reply:
Updated to this, thanks